app/views/manage_assessments/coverages/new.html.erb

```diff
@@ -1,3 +1,15 @@
+<%= content_for :full_bleed do %>
+  <div class="madlib-header">
+    <h1 class="headline-narrow">
+      Add a class
+    </h1>
+    <h2 class="headline-narrow-subhead">
+      <%= @coverage.course.number %> &middot;
```

Comment:
Maybe move the `&middot;` onto the next line? Not a huge deal, but then you can see all the pieces.